sources/packages/web/src/types/contracts/StudentContract.ts

```diff
@@ -49,3 +54,8 @@ export interface StudentProfile {
   country: string;
   postalCode: string;
 }
+
+export interface StudentFormInfo extends StudentInfo {
+  birthDateFormatted: string;
```

Comment:
Can you please elaborate on where this was needed for this particular ticket?

Reply:
https://github.com/bcgov/SIMS/blob/main/sources/packages/web/src/views/student/Student.vue#L200 is calling https://github.com/bcgov/SIMS/blob/main/sources/packages/web/src/services/StudentService.ts#L42 (this code was already there without interface, I just added interface, since it was throwing me errors)